src/rocksdb_options.rs

```diff
@@ -740,6 +740,12 @@ impl DBOptions {
         }
     }
 
+    pub fn set_fifo_compaction_options(&mut self, fifo_opts: &FifoCompactionOptions) {
```

Comment:
why not moving FifoCompactionOptions here directly?

Reply:
I changed it to a move. I think I tried to follow the pattern from BlockBasedOptions.